.github/workflows/kotlin.yml

```diff
@@ -78,29 +29,20 @@ jobs:
           - detekt
           # Build the JMH benchmarks to verify, but don't run them.
           - jmhJar
+
     steps:
       - uses: actions/checkout@v2
       - name: set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
 
-      ## Caching
-      - name: Cache build artifacts
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.GRADLE_HOME }}/caches
-          # Don't set restore-keys so cache is always only valid for the current build config.
-          # Also don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
-          key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
-
       ## Actual task
       - name: Check with Gradle
-        run: ./gradlew ${{ matrix.gradle-task }} --build-cache --no-daemon --stacktrace --gradle-user-home "$GRADLE_HOME"
+        run: ./gradlew ${{ matrix.gradle-task }} --no-daemon --stacktrace --gradle-user-home "$GRADLE_HOME"
```

Comment:
I think you can drop the `--gradle-user-home` args too, and get rid of that envar definition at the top of the file.